userguide/tn_initial.rst

```diff
@@ -274,17 +274,18 @@ Enter the default password of *abcd1234*.
    :guilabel:`Password confirmation` fields, and click :guilabel:`OK`
    to save the new password to use on subsequent logins.
 
-On the first login, the EULA found in :ref:`Appendix A` is displayed,
-along with a box where the license key for the %brand% array can be
-pasted. Read the EULA, paste in the license key, then click
-:guilabel:`I agree` to accept the EULA and apply the license.
+On the first login, the End User License Agreement (EULA) found in
+:ref:`Appendix A` is displayed, along with a box where the license key
+for the %brand% array can be pasted. Read the EULA, paste in the license
```

Comment:
```suggestion
for the %brand% array is pasted. Read the EULA, paste in the license
```

Reply:
The tense in the suggested version is not quite right, because when the user sees that field, the key is not yet there.  We tell them to paste the key in the next sentence, so how about just:
`` ...along with a box for the license key. Read the EULA, paste in ...``